charts/README.md

```diff
@@ -19,6 +19,19 @@
 helm repo add azuredisk-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/charts
 helm install azuredisk-csi-driver azuredisk-csi-driver/azuredisk-csi-driver --namespace kube-system
 ```
+## install Azure Disk CSI Driver V2 (Preview)
+Applicable to any K8s cluster without the Azure Disk CSI Driver V1 installed. If V1 is installed, proceed to side-by-side installation instructions below. The V1 driver is installed by default on K8s versions 1.21 and above.   
```

Comment:
"The V1 driver is installed by default in AKS clusters with Kubernetes version 1.21 and later."

You should also spell out Kubernetes in official documentation rather than abbreviate.